src/tests/bigint_test.c

```diff
@@ -227,6 +227,7 @@ void bigint_mod_exp_sample ( const bigint_element_t *base0,
 	DBG_HDA ( 0, &value_temp, sizeof ( value_temp ) );		\
 	bigint_add ( &addend_temp, &value_temp );			\
 	DBG_HDA ( 0, &value_temp, sizeof ( value_temp ) );		\
+	memset( result_raw, 0, sizeof ( result_raw ) );                 \
```

Comment:
This issue was fixed yesterday in https://github.com/ipxe/ipxe/commit/f3f568e38

As a general rule, iPXE actively avoids performing this kind of unnecessary initialisation of values, because that prevents the compiler from warning about real uninitialised-value situations.